lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -107,6 +107,10 @@ function dosomething_helpers_preprocess_partners_vars(&$vars) {
       if (!empty($image_node->field_image_landscape)) {
         $img = dosomething_image_get_themed_image($image_node->nid, 'landscape', '550x300');
         $vars['partner_info'][$delta]['image'] = $img;
+      } else {
```

Comment:
OK, I'll check other function calls and will refactor `dosomething_image_get_themed_image()` to provide default image as a fallback.